modules/monitoring-the-training-job.adoc

```diff
@@ -0,0 +1,92 @@
+The example training job in this section is based on the IBM and Hugging Face tuning example provided link:https://github.com/foundation-model-stack/fms-hf-tuning/tree/main/examples/prompt_tuning_twitter_complaints[here]. 
+
+
+.Prerequisites
+ifdef::upstream,self-managed[]
+* You have logged in to {openshift-platform} with the `cluster-admin` role.
+endif::[]
+ifdef::cloud-service[]
+* You have logged in to OpenShift with the `cluster-admin` role.
+endif::[]
+
+ifndef::upstream[]
+* You have access to a data science cluster that is configured to run distributed workloads as described in link:{rhoaidocshome}{default-format-url}/working_with_distributed_workloads/configuring-distributed-workloads_distributed-workloads[Configuring distributed workloads].
+endif::[]
+ifdef::upstream[]
+* You have access to a data science cluster that is configured to run distributed workloads as described in link:{odhdocshome}/working-with-distributed-workloads/#configuring-distributed-workloads_distributed-workloads[Configuring distributed workloads].
+endif::[]
+
+ifndef::upstream[]
+* You have created a data science project. 
+For information about how to create a project, see link:{rhoaidocshome}{default-format-url}/working_on_data_science_projects/working-on-data-science-projects_nb-server#creating-a-data-science-project_nb-server[Creating a data science project].
+endif::[]
+ifdef::upstream[]
+* You have created a data science project. 
+For information about how to create a project, see link:{odhdocshome}/working-on-data-science-projects/#_using_data_science_projects[Creating a data science project].
+endif::[]
+
+* You have Admin access for the data science project.
+** If you created the project, you automatically have Admin access. 
+** If you did not create the project, your cluster administrator must give you Admin access.
+
+* You have access to a model.
+* You have access to data to train the model.
+
+ifndef::upstream[]
+* You are running the training job as described in link:{rhoaidocshome}{default-format-url}/working_with_distributed_workloads/tuning-a-model-by-using-the-training-operator_distributed-workloads#running-the-training-job_distributed-workloads[Running the training job].
+endif::[]
+ifdef::upstream[]
+* You are running the training job as described in link:{odhdocshome}/working-with-distributed-workloads/#running-the-training-job_distributed-workloads[Running the training job].
+endif::[]
+
+.Procedure
+ifdef::upstream,self-managed[]
+. In the {openshift-platform} console, select your project from the *Project* list. 
+endif::[]
+ifdef::cloud-service[]
+. In the OpenShift console, select your project from the *Project* list.
+endif::[]
+. Click *Workloads* -> *Pods*.
+
+. Search for the pod that corresponds to the PyTorch job.
+The default pod name is *kfto-demo-master-0*, where `kfto` is the name of the project where the job is running.
+Replace the example namespace value `kfto` with the name of your project. 
```

Comment:
should this be a sub-step?

Reply:
This text is not a step, it's explaining how to find the pod. I've rephrased to make it clearer.